types/kendo-ui/index.d.ts

```diff
@@ -4821,6 +4931,9 @@ declare namespace kendo.ui {
     }
 
     interface GridColumn {
+		dataTextField?: string;
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/kendo-ui/index.d.ts;
  Line: 4934;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.